plugins/node/opentelemetry-instrumentation-restify/src/types.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Span } from '@opentelemetry/api';
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+
+export interface RestifyRequestInfo {
+  request: any; // RestifyRequest object from restify package
```

Comment:
What is `RestifyRequest`? cannot find it in `@types/restify`.
Did you mean [`Request`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9b1c89f2a91f76ea076e9bbec5131022fd6bdbb2/types/restify/index.d.ts#L434)?

Reply:
Yes, I think that's the one. It's reported as `IncomingMessage` when I log it, but I think it's that Request type.